project/src/index.tsx

```diff
@@ -2,8 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/app/app';
 
+const numberOfAds = {
+  ADS: 10,
+};
```

Comment:
Пока мне кажется излишним создавать целый объект для одного значения используй просто константу. 
```suggestion
const NUMBER_OF_ADS = 10;
```